templates/invoice/pdf_invoice.html.twig

```diff
@@ -208,6 +208,21 @@
             </td>
         </tr>
     </table>
+    <br>
+    <div style="width: 100%; padding-left: 32px">
+        {% if invoice.typePayment == 'Carte' %}
+            <p>Le type de paiement du client est par {{ invoice.typePayment }}</p>
+        {% endif %}
+
+        {% if invoice.typePayment == 'Chèque' %}
+            <p>Le type de paiement du client est par {{ invoice.typePayment }}</p>
+        {% endif %}
+
+        {% if invoice.typePayment == 'Espèce' %}
+            <p>Le type de paiement du client est en {{ invoice.typePayment }}</p>
+        {% endif %}
```

Comment:
Since the difference it's only "en" and "par" we should just call the sentence once and make an if statement to render "en" or "par"